frontend/app/models/award-store/award-store.ts

```diff
@@ -0,0 +1,48 @@
+import { Instance, SnapshotOut, types } from "mobx-state-tree"
+import { withEnvironment } from "../extensions/with-environment";
+
+
+/**
+ * Model description here for TypeScript hints.
+ */
+export const AwardStoreModel = types
+  .model("AwardStore")
+  .props({
+    awards: types.optional(types.array(types.array(types.string)), [["No Awards Yet ...", "Keep completing goals to get one soon!"]]),
+  })
+  .views(self => ({})) // eslint-disable-line @typescript-eslint/no-unused-vars
+  .extend(withEnvironment)
+  .actions(self => ({
+    setAwards: (AwardsList) => {
+      if (AwardsList) {
+        __DEV__ && console.log("Setting Awards list " + AwardsList.toString());
+        self.awards = AwardsList;
+      } else { __DEV__ && console.log("Unsetting Awards list"); }
+    },
+  })).actions( self => ({
+    getAllAwards: () => {
+      return self.environment.api.getAllAwards().then(res => {
```

Comment:
I think we decided that the awards would just be a frontend thing for now, correct?

Reply:
yes